### ISSUE TITLE
Clarification about the modes

### ISSUE THREAD

**Issue by fatbao55** [19reborn/NeuS2#4]
Hi authors, 

I would like to clarify what does the different modes refer to? Does mode==nerf` refers to your implementation of NeuS2, and does `mode==sdf` refers to the implementation by [Instant-NGP](https://github.com/NVlabs/instant-ngp)?

**Comment by 19reborn** [comment]
Yes, that's correct. We only check the codes for NeuS2 and will remove other modes. Apologies for any confusion caused.

**Comment by fatbao55** [comment]
@19reborn No worries, thanks for the clarification. For the results of Instant-NGP in your paper, is it the NeRF version or SDF version?

**Comment by 19reborn** [comment]
@fatbao55 It's the NeRF version for Instant-NGP.

**Comment by fatbao55** [comment]
@19reborn Thanks for clarifying. I was wondering what are the differences between NeuS2 and the SDF version of Instant-NGP? I'm not too familiar, but both seems to be implementations of SDF with the exception that NeuS incorporates foreground masks.

**Comment by 19reborn** [comment]
@fatbao55 the SDF version of Instant-NGP is not the same task with NeuS2.  It learns SDFs from point clouds rather than images.
